lib/modules/job/providers/jobs.provider.dart

```diff
@@ -9,23 +11,52 @@ class JobsProvider extends AppProvider {
   final _jobService = inject.get<JobService>();
 
   List<Job> _jobs = [];
+
   bool _fetchLoading = false;
   dynamic _fetchError;
 
   bool get fetchLoading => _fetchLoading;
 
+  BaseException? get fetchError => _fetchError;
```

Comment:
fa erorile dynamic type in loc de BaseException

Daca cumva crapa aplicatia la json parsing nu o sa se arunce baseException si o sa ne fie mai greu sa depistam problema

Reply:
Am schimbat în ec14c33